tsconfig.json

```diff
@@ -1,6 +1,7 @@
 {
+    "extends": "@tsconfig/node18/tsconfig.json",
     "compilerOptions": {
-        "strict": true
+        "lib": ["DOM"]
     },
     "typedocOptions": {
         "entryPoints": ["pkg/matrix_sdk_crypto_wasm.d.ts"],
```

Comment:
this presumably needs an update, to make sure that initAsync continues to be documented?

Reply:
Good point, fixed in [b89ec58](https://github.com/matrix-org/matrix-rust-sdk-crypto-wasm/pull/167/commits/b89ec58f9b767d047fa3312a72c2382ca6995641)